.eslintrc.js

```diff
@@ -16,7 +17,7 @@ module.exports = {
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "plugins": ["lit", "lit-a11y"],
+  "plugins": ["lit", "lit-a11y", "jsdoc"],
```

Comment:
This adds the plugin, but we aren't currently using any of the rules. I'm currently going through the rules to try to get a similar config to earlier. The recommended setup seems too restrictive.

Reply:
Good catch... I forgot to mention that I didn't add the [rules](https://github.com/gajus/eslint-plugin-jsdoc#configuration). Assumed that this would be part of the conversation.

Reply:
The recommend set seems to be doing what we have been living with for a while now?

Reply:
No, it adds [a lot more](https://github.com/gajus/eslint-plugin-jsdoc#configuration). There isn't a 1-1 mapping between the deprecated eslint rules and the new plugin.

Reply:
Suggested rules:

```
"jsdoc/require-jsdoc": "error",
"jsdoc/require-param-description": "error",
"jsdoc/require-param-type": "error",
"jsdoc/require-param":  ["error", { "enableFixer": false } ],
"jsdoc/require-returns-check": "error",
"jsdoc/require-returns": "error",
```

This gets us pretty close to the previous [valid-jsdoc](https://eslint.org/docs/rules/valid-jsdoc) eslint rule. I tested the config out on my auro-interruption branch and the only change was adding a missing parameter description to a method.